tests/test_itrf_converter.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import pandas as pd
+
+from iceflow.itrf.converter import _datetime_to_decimal_year, transform_itrf
+
+
+def test_transform_itrf():
+    synth_df = pd.DataFrame(
+        {
+            # Note: duplicate data here because otherwise a deprecation warning
+            # error about single-value series is raised from pandas' internals,
+            # and pytest complains.
+            "latitude": [70, 70],
+            "longitude": [-50, -50],
+            "elevation": [1, 1],
+            "ITRF": ["ITRF93", "ITRF93"],
+        },
+    )
+
+    constant_datetime = pd.to_datetime("1993-07-02 12:00:00")
+    synth_df.index = pd.Index([constant_datetime] * 2, name="utc_datetime")
+
+    result = transform_itrf(
+        data=synth_df,
+        target_itrf="ITRF2014",
+    )
+
+    # Before and after values were extracted from the synthetic dataset in the
+    # ITRF corrections notebook put together by Kevin. See:
+    # https://github.com/nsidc/NSIDC-Data-Tutorials/blob/a35203f18841456d258fac3a11dc04f44f839d9d/notebooks/iceflow/corrections.ipynb
+    assert result.latitude.to_numpy()[0] == 69.99999974953995
+    assert result.longitude.to_numpy()[0] == -50.0000001319163
+    assert result.elevation.to_numpy()[0] == 1.0052761882543564
+
+
+def test__datetime_to_decimal_year():
```

Comment:
Maybe we could mock the local time zone to a couple different values to detect regressions!

Reply:
This is a good idea but I'm struggling to figure out how to mock the TZ. I can manually change the TZ by setting the envvar at runtime (e.g., `TZ=UTC inv test.pytest`), but mocking the envvar in Python fails. I've tried:

* `os.environ["TZ"] = "UTC"`
* `monkeypatch.setenv("TZ", "UTC")`

I've also tried calling `time.tzset()` after doing the above.

Reply:
I just kind of assumed there was a clean solution to this :laughing: Feels weird that there isn't!

Reply:
Oh dear, I think I'm just tired this morning. It is working...commit incoming.

Reply:
Oh sick!

Reply:
8b31da5